core/Plugin/Manager.php

```diff
@@ -1191,6 +1191,8 @@ private function executePluginDeactivate($pluginName)
      */
     private function unloadPluginFromMemory($pluginName)
     {
+        $this->unloadPlugin($pluginName);
+
```

Comment:
I was wondering about `Plugin\Manager::unloadPluginFromMemory()` and `Plugin\Manager::unloadPlugin()` (both in the plugin manager)